docs/content/users/quickstart.md

```diff
@@ -366,14 +366,14 @@ Start a new [Kirby CMS](https://getkirby.com) project or use an existing one.
 
 Use a new or existing Composer project, or clone a Git repository.
 
-The Laravel project type can be used for [Lumen](https://lumen.laravel.com/) like it can for Laravel. DDEV automatically updates or creates the `.env` file with the database information.
+The Laravel project type can be used for Laravel Livewire, Inertia and others like it can for Laravel. DDEV automatically updates or creates the `.env` file with the database information.
```

Comment:
This would be better with links to Livewire, Inertia, etc.

Reply:
Added links to https://laravel.com/docs/11.x/starter-kits and https://livewire.laravel.com/, inertia is not so well know as far as I know. 👍